Android.mk

```diff
@@ -136,6 +145,12 @@ ifeq ($(SUPPORT_WAVPACK),true)
     LOCAL_STATIC_LIBRARIES += wavpack
 endif
 
+ifeq ($(SUPPORT_GME),true)
+    LOCAL_CFLAGS += -DMUSIC_GME
+    LOCAL_C_INCLUDES += $(LOCAL_PATH)/$(GME_LIBRARY_PATH)
```

Comment:
Do you not need to append `/gme` here ?

Reply:
SDL_mixer source includes it like `#include <gme/gme.h>`, so it's correct include path.

Reply:
Ah yes. Sorry for the noise.